packages/react-select/src/Select.js

```diff
@@ -1815,10 +1868,14 @@ export default class Select extends Component<Props, State> {
   renderLiveRegion() {
```

Comment:
It might be good to abstract this to a separate component at this time. What do you think?

Reply:
Thanks for your feedback!
I am not sure. Looking at other render helpers, `renderLiveRegion` seems to line up pretty well? On the other hand one could argue for a decoupling exercise of the whole `Select` class, being ~1600 lines tall. But perhaps that work should be logged in a separate work package. What are you thoughts?

Reply:
I agree! You are following the convent. Also agree that this is 1600 lines. So maybe another PR would be good to refactor some of the render* function into separate components.